diff --git a/api-gateway-service/src/main/java/com/comment/insight/api/gateway/controller/FallbackController.java b/api-gateway-service/src/main/java/com/comment/insight/api/gateway/controller/FallbackController.java
index 4f865b0..1d53c5d 100644
--- a/api-gateway-service/src/main/java/com/comment/insight/api/gateway/controller/FallbackController.java
+++ b/api-gateway-service/src/main/java/com/comment/insight/api/gateway/controller/FallbackController.java
@@ -9,7 +9,7 @@ import reactor.core.publisher.Mono;
 @RequestMapping("/api/fallback")
 public class FallbackController {
 
-    @GetMapping("/connector-service")
+    @RequestMapping("/connector-service")
     @ResponseStatus(HttpStatus.SERVICE_UNAVAILABLE)
     public Mono<ErrorResponse> connectorServiceFallback() {
         return Mono.just(new ErrorResponse(
@@ -20,7 +20,7 @@ public class FallbackController {
         ));
     }
 
-    @GetMapping("/youtube-connector-service")
+    @RequestMapping("/youtube-connector-service")
     @ResponseStatus(HttpStatus.SERVICE_UNAVAILABLE)
     public Mono<ErrorResponse> youtubeConnectorServiceFallback() {
         return Mono.just(new ErrorResponse(
@@ -30,4 +30,15 @@ public class FallbackController {
                 "/api/youtube"
         ));
     }
+
+    @RequestMapping("/sentiment-service")
+    @ResponseStatus(HttpStatus.SERVICE_UNAVAILABLE)
+    public Mono<ErrorResponse> sentimentServiceFallback() {
+        return Mono.just(new ErrorResponse(
+                HttpStatus.SERVICE_UNAVAILABLE.value(),
+                "Service Unavailable",
+                "Sentiment Service is temporarily unavailable. Please try again later.",
+                "/api/sentiment"
+        ));
+    }
 }
